smart-contracts/contracts/mocks/TimeMachineMock.sol

```diff
@@ -0,0 +1,26 @@
+pragma solidity ^0.5.0;
+
+import '../PublicLock.sol';
+
+contract TimeMachineMock is
+  PublicLock
+{
+  /**
+  * @notice Modify the expirationTimestamp of a key
```

Comment:
Minor: but maybe let's not duplicate comments too much.  They are likely to get out of sync as we make changes.  Since this is just a test artifact I'd say either drop the comment or only comment on the mock specific side of things here.

Reply:
done.